app/models/game.rb

```diff
@@ -23,7 +23,6 @@ def reset_piece_players
     pieces.where(is_black: false).update_all(user_id: user_id_white)
   end
 
-  # rubocop:disable Metrics/AbcSize
```

Comment:
Right, so `Metrics/AbcSize` shouldn't be disabled in `.rubocop.yml` but that line above should appear before this method:

```ruby
# rubocop:disable Metrics/AbcSize
```